input_topic: zed/point_cloud/cloud_registered
output_topic: scene_filtered/pointcloud